fluture.js

```diff
@@ -1565,6 +1606,22 @@
     return `${this._left.toString()}.finally(${this._right.toString()})`;
   };
 
+  //----------
+
+  function FutureNever(){}
+
+  FutureNever.prototype = Object.create(Future.prototype);
+
+  FutureNever.prototype._f = function FutureNever$fork(){
+    return noop;
+  };
+
+  FutureNever.prototype.toString = function FutureNever$toString(){
+    return `Future.never`;
+  };
+
+  Future.never = new FutureNever;
```

Comment:
There's no reason for it. But I guess it's not really important. I think it's fine like this.